samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -171,6 +171,35 @@
     }
   },
 
+  //
+  // ChatSkill prompts are used to generate responses to user messages.
+  // - CompletionTokenLimit is the token limit of the chat model, see https://platform.openai.com/docs/models/overview
```

Comment:
nit: might be worth adding "Adjust the limit according to the completion model you select."